R/bpvalidate.R

```diff
@@ -124,6 +124,10 @@ setMethod("show", "BPValidate", function(object) {
         i <- vapply(unknown, function(x) {
             !exists(x, envir = env, inherits = FALSE)
         }, logical(1))
+        ## Force evaluation of the known arguments to
+        ## make sure they will be exported correctly
+        known <- unknown[-i]
+        lapply(known, function(x) force(env[[x]]))
```

Comment:
Since the return value is not used, does it make better sense to use a `for()` loop?

Reply:
I also thought about this. Here is my blind spot. Will a for-loop be more efficient than an apply function? 

I know in practice this may not make any visible difference, but I was always told that we should avoid using for-loop if possible. Therefore, I chose `lapply` as it is more concise than the for-loop.

Reply:
Thanks for the explanation. That makes much sense. I have updated the code to use the for loop.